### PR TITLE
Implement GetForkSchedule in the config API

### PR DESCRIPTION
**Pull request by rkapka**
**What type of PR is this?**

Feature

What does this PR do? Why is it needed?

This PR implements config API's `GetForkSchedule` method according to the spec: https://ethereum.github.io/eth2.0-APIs/#/Config/getForkSchedule

Which issues(s) does this PR fix?

Part of #7510

Other notes for review

N/A

### CODE REVIEW

File: beacon-chain/rpc/beaconv1/config.go [hunk 1]
```diff
@@ -35,3 +65,14 @@ func (bs *Server) GetDepositContract(ctx context.Context, req *ptypes.Empty) (*e
...
 }
+
+func sortedEpochs(forkSchedule map[uint64][]byte) []uint64 {
+	sortedEpochs := make([]uint64, len(forkSchedule))
+	i := 0
+	for k := range forkSchedule {
+		sortedEpochs[i] = k
+		i++
+	}
+	sort.Slice(sortedEpochs, func(i, j int) bool { return sortedEpochs[i] < sortedEpochs[j] })
```
Comment (rauljordan) [inline beacon-chain/rpc/beaconv1/config.go:76]: Shadowing here?

File: beacon-chain/rpc/beaconv1/config.go [hunk 2]
```diff
@@ -11,8 +12,37 @@ import (
...
+	if len(schedule) == 0 {
+		return &ethpb.ForkScheduleResponse{
+			Data: make([]*ethpb.Fork, 0),
+		}, nil
+	}
+
+	epochs := sortedEpochs(schedule)
+	forks := make([]*ethpb.Fork, len(schedule))
+	var previous, current []byte
+	for i, e := range epochs {
```
Comment (terencechain) [inline beacon-chain/rpc/beaconv1/config.go:29]: This assumes sequentiality. What about?

Screen Shot 2021-01-27 at 7 44 01 AM

I don't think `ForkVersionSchedule` in config is spec'ed correctly. It should be contain both current and previous versions

Comment (rkapka) [reply beacon-chain/rpc/beaconv1/config.go:29]: The forks in the config are scheduled forks. Isn't it reasonable to assume sequentiality?

Comment (terencechain) [reply beacon-chain/rpc/beaconv1/config.go:29]: What if client decides to jump between forks? Do we just delete the history in this regard? 

I'm fine with that tbh. Let's assume sequentiality unless something else comes up. I dont want to over engineer this problem

**Review by rauljordan — APPROVED** [review]

**Pull request merged**